integration_tests/book_gorm_redis/test_starter.go

```diff
@@ -0,0 +1,95 @@
+package book_gorm_redis
+
+import (
+	"context"
+	redis2 "github.com/go-redis/redis"
```

Comment:
please avoid use default alias like `redis2`

Reply:
some of these are needed as I already have a `redis` package coming from somewhere else - I'll update the name though :)